packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -15,13 +15,15 @@ export type AnnualHighlightCardsProps = {
 	};
 	titleHref?: string | null;
 	year?: string | number | null;
+	navigation?: object | null;
```

Comment:
```suggestion
	navigation?: React.ReactNode;
```

Is the type `React.ReactNode` more accurate?